core/README.md

```diff
@@ -97,22 +97,23 @@ Using a specific script (just through `docker compose` CLI and a set of docker-c
 allows to run a single core worker for all infra on localhost network:
 
 ```sh
-./scripts/single-worker-compose.sh up -d
+# run all services
+./scripts/single-worker-compose.sh --profile "*" up -d
 
 # or exclude 'core' service straight away:
-./scripts/single-worker-compose.sh up -d --scale core=0
+./scripts/single-worker-compose.sh --profile with-editoast up -d
```

Comment:
Might be worth to add the `with-core` option as an example, since they are the only 2 services with this option, what do you think?

Reply:
Indeed, more explicit readme pushed after your suggestion: https://github.com/OpenRailAssociation/osrd/compare/64521221fc435ad22284e8073b3c469b7fe60320..5d67e80576cc800d26f8271ecb42d0ba4e1cf1e1#diff-ea08f26e820e40e902af500eb960c815bbc5b5f23bacdc9e884cee42374024cb
Note that `--profile with-editoast --profile with-core` == `--profile "*"`.